### PR TITLE
Basic command line argument handling for the server.

### PR DESCRIPTION
**Pull request by PJB3005**
Also removes the NuGet packages from the repo.

### CODE REVIEW

**Comment by d34d10cc** [comment]
You monster.

File: SS14.Server/Program.cs [hunk 1]
```diff
@@ -37,17 +34,14 @@ private static void Main(string[] args)
...
-                switch(args[i])
-                {
-                    case "--fulldump":
-                        fullDump = true;
-                        break;
-                }
-            }
+            var options = new CommandLineArgs();
+            bool result = CommandLine.Parser.Default.ParseArguments(args, options);
+            if (!result)
```
Comment (ComicIronic) [inline SS14.Server/Program.cs]: Yech. Blech. Why.

**Comment by ComicIronic** [comment]
You a real one. Let's see this break something package-wise.

**Comment by PJB3005** [comment]
People might have to restore packages but that should be default in Visual Studio IIRC. Even if it isn't it's a simple context menu option I'm pretty sure.

**Pull request merged**